core/storage/concept/persisted_map.hpp

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_PERSISTED_MAP_HPP
+#define KAGOME_PERSISTED_MAP_HPP
+
+#include <boost/filesystem/path.hpp>
```

Comment:
Are you including it, so the implementation is not needed to do it? Anyway, looks like a bad idea - what if the implementation would use some other way to define path?